experimental/Schemes/SimplifiedSpec.jl

```diff
@@ -171,6 +172,20 @@ function _flatten_open_subscheme(
   return _flatten_open_subscheme(W, C, iso=new_iso)
 end
 
+function _flatten_open_subscheme(
```

Comment:
Please, have a look at line 106.

Reply:
Okay great. I see that you document some private functions with 
`#comments`. This is good. 
A suggestion you may ignore:
Turn some comments into docstrings. (Since they are private I am okay with ignoring docstring conventions.) Then one can also ask for the docu in the repl. 
Sometimes I use private methods for my purposes (at my own risk). If they are documented, it is helpful.